qualtran/_infra/Bloqs-Tutorial.ipynb

```diff
@@ -83,29 +109,46 @@
   {
    "cell_type": "markdown",
    "id": "c632807b",
-   "metadata": {},
+   "metadata": {
+    "id": "c632807b"
+   },
    "source": [
-    "There is only one mandatory method we must implement to have a well-formed `Bloq`. There\n",
-    "are many other methods we can optionally implement to encode more information about the\n",
-    "bloq, which we will add as we go along.\n",
+    "There is only one mandatory method we must implement to have a well-formed `Bloq`.\n",
     "\n",
     "The mandatory method is the `Bloq.signature` property. This declares what the inputs and\n",
     "outputs are for our bloq, and is a list of registers. A register has a name and quantum-type\n",
     "information. By default, a register declares both an input and an output allowing quantum\n",
-    "data to pass through it, like the \"control\" register below. We call these `THRU` registers.\n",
-    "\n",
-    "The `Bloq.signature` property can be thought of as analogous\n",
-    "to the function signature in ordinary programming. You can think of a bloq\n",
-    "with just this property implemented like a function\n",
-    "declarations in a C header (`*.h`) file."
+    "data to pass through it, like the \"control\" register below. We call these `THRU` registers.\n"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "id": "22d8749d",
-   "metadata": {},
-   "outputs": [],
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-06-11T03:34:54.004785Z",
+     "start_time": "2024-06-11T03:34:51.338589Z"
+    },
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 351
+    },
+    "id": "22d8749d",
+    "outputId": "b4764180-e624-4579-9e94-f7dc277f374b"
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Register(name='control', dtype=QBit(), _shape=(), side=<Side.THRU: 3>)"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
```

Comment:
can you "clear output of all cells" before committing these notebooks so the diff is clearer and we don't clutter the git history

Reply:
Thanks. I will do that

Reply:
@mpharrigan I have removed output of all cell and the outdated table of contents in all-in-one tutorial.